x/emissions/client/cli/tx_create_withdraw_emssions.go

```diff
@@ -0,0 +1,37 @@
+package cli
+
+import (
+	"errors"
+
+	sdkmath "cosmossdk.io/math"
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/client/flags"
+	"github.com/cosmos/cosmos-sdk/client/tx"
+	"github.com/spf13/cobra"
+	"github.com/zeta-chain/zetacore/x/emissions/types"
+)
+
+func CmdCreateWithdrawEmission() *cobra.Command {
```

Comment:
Why "create"?

We should name the CLI as the action of the message `WithdrawEmission`

Reply:
f2d4d84768a74f837eece318f4b2c9fa557c3f81